### ISSUE TITLE
A11y_Azure Tools for VS Code [BM]_Default Screen_Screen Reader: NVDA/Narrator is not reading the star rating information properly. It is just reading how many  users rated it.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/vscode-azurefunctions#2671]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### Actual Result:
NVDA/Narrator is not reading the star rating information properly. It is just reading how users rated it.

### Expected Result:
NVDA/Narrator  should read the rating information(eg. 4 star taring) along with number of users rated.

### GitHubTags:
#Azure Tools for VS Code; #A11ySev1; #A11yTCS; #AILimited; #BM-Azure Tools for VS Code-Win-Feb2021; #DesktopApp; #Screen Reader; #WCAG1.3.1; #NVDA; #Win10; #Narrator; #Element:Link;

### OS Details:
Windows 10 Version 20H2 (OS Build 19042.685)

### Environment Details:
Application Name: Azure Tools for VS Code
Version: 1.53.1
Commit: 5d424b828ada08e1eb9f95d6cb41120234ef57
Electron: 11.2.1
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Windows_NT x64 10.0.19042
Azure Function Version: 1.2.1

### Repro Steps:

1. Open Visual studio code
2. Turn on NVDA/Narrator
2. Tab till extension button and press enter on it.
3. Search for "Azure Functions" and select it.
3. Tab till "Install" button and press enter on it.
4. Tab till "Azure" button from left side pane and press enter on it.
4. Default screen will get open.
5. Tab till "rating" link and verify how NVDA/Narrator is reading it.

### User Impact:
Assistive technology users will not get complete information about rating if screen reader is not reading it.

### Attachment for Reference:

[Uploading 2671_NVDA issue_Default Screen.zip…]()
[2671_Narrator issue.zip](https://github.com/microsoft/vscode-azurefunctions/files/6020071/2671_Narrator.issue.zip)

**Comment by ejizba** [comment]
Our extension does not own or control the marketplace. This would reproduce for any VS Code extension and should be filed directly on VS Code instead: https://github.com/microsoft/vscode/issues
